### PR TITLE
fix: add Ankan Bag

### PR DESCRIPTION
**Pull request by ankanbag101**

### CODE REVIEW

File: public/data/ankanbag101.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/ankan-bag-74a238222/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Email me",
+        "url": "mailto:bagankan@gmail.com"
```
Comment (emmalearnscode) [inline public/data/ankanbag101.json:24]: Do you not want to choose an icon for this one? We have an envelope

Comment (ankanbag101) [reply public/data/ankanbag101.json:24]: Thanks, I will add the envelope

**Review by emmalearnscode — APPROVED** [review]

File: public/data/ankanbag101.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      },
+      {
+        "title": "Won first prize in web-battle-hackathon",
+        "date": "October 2021",
+        "icon": "dollar",
+        "color": "green",
+        "description": "First time participated on a hackathon and won first prize of 5000 rupees"
+      },
+      {
+        "title": "published first article on Medium",
```
Comment (syedsohan) [inline public/data/ankanbag101.json:45]:
```suggestion
        "title": "Published first article on Medium",
```

File: public/data/ankanbag101.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        "icon": "dollar",
+        "color": "red",
+        "description": "Won $100 in the Hackathon organized by Reskill and Spark AR on creating AR effects. Was placed in Top 10 out of 2000+ applications."
+      },
+      {
+        "title": "First Contribution to Open-Source",
+        "date": "December 2021",
+        "icon": "github",
+        "color": "grey",
+        "description": "Contributed to Daily.dev readme file"
```
Comment (syedsohan) [inline public/data/ankanbag101.json:64]:
```suggestion
        "description": "Contributed to Daily.dev README file"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/ankanbag101.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
+{
+    "name": "Ankan Bag",
+    "type": "personal",
+    "bio": "An alien👽 trying to make a difference in the tech world🔥",
```
Comment (Panquesito7) [inline public/data/ankanbag101.json:4]:
```suggestion
    "bio": "An alien 👽 trying to make a difference in the tech world 🔥",
```

File: public/data/ankanbag101.json (hunk 3 above)
Comment (ankanbag101) [reply public/data/ankanbag101.json:64]: Thanks for the suggestion, I will change it.

File: public/data/ankanbag101.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      },
+      {
+        "name": "Email me",
+        "url": "mailto:bagankan@gmail.com",
+        "icon": "envelope"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started GitHub account",
```
Comment (krishguptadev) [inline public/data/ankanbag101.json:30]: I am not sure if you **Start** github accounts

```suggestion
        "title": "Created GitHub account",
```

**Review by syedsohan — APPROVED** [review]

File: public/data/ankanbag101.json (hunk 5 above)
Comment (emmalearnscode) [reply public/data/ankanbag101.json:30]: Yes, started works just fine here. No need to change.

**Review by syedsohan — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**